### PR TITLE
Reworking how things work after a Rawls update.

### PR DESCRIPTION
**Pull request by MattPutnam**
NOTE: This code is dependent on PR#50 in orchestration.

### CODE REVIEW

**Comment by eddiebroad** [comment]
From this commit, should there be seen any difference from the user perspective (in the browser)?  I didn't see any difference from that perspective.

**Comment by eddiebroad** [comment]
PR 50 of the orchestration was happily merged around 1:58PM this afternoon!

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs [hunk 1]
```diff
@@ -28,6 +28,8 @@
        :methodStoreConfig {:methodConfigNamespace (str "msc_ns_" (inc i))
                            :methodConfigName (str "msc_n_" (inc i))
                            :methodConfigVersion (str "msc_v_" (inc i))}
+       ;; Real data doesn't have the following fields, but for mock data we carry the same
+       ;; objects around, so initialize them here for convenience
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs]: Nice comment.

**Comment by dmohs** [comment]
:+1:

**Comment by MattPutnam** [comment]
@eddiebroad The behavior is slightly different.  The method config edit page now has to load the config from Rawls, instead of just using the same object from the table in the method config tab.

**Comment by eddiebroad** [comment]
@MattPutnam  (2-3 min before this comment) it looks like lines 220-228 do that? is that right?

**Comment by MattPutnam** [comment]
Right.

**Comment by eddiebroad** [comment]
:+1:

**Comment by eddiebroad** [comment]
i don't see why this shouldn't be merged

**Pull request merged**